src/game.cpp

```diff
@@ -4946,17 +4946,16 @@ void Game::playerInviteToParty(uint32_t playerId, uint32_t invitedId)
 		return;
 	}
 
-	Party* party = player->getParty();
+	auto party = player->getParty();
```

Comment:
in this case I think Party_ptr would be better for readability

Reply:
Are you sure you prefer `Party_ptr party = player->getParty()`? :thinking: fiesta